### DISCUSSION TITLE
["Fac Simile" 27/10/2022] - Cache

### DISCUSSION THREAD

**Discussion by alem1105** [CS-Swap/Architettura-degli-Elaboratori#12] [Cache]
### Traccia Esercizio

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/28d62b3e-140d-4e36-8d0c-3510c7008558)

### Soluzione Prof

<details><summary>Soluzione</summary>
<p>

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/c56629d2-2b27-4986-b122-6238b7ea1d9b)

</p>
</details>

**Comment by thegeek-sys** [comment]
<img width="1087" alt="Screenshot 2024-06-04 alle 09 34 12" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/63b8eb74-0fc9-41d8-93b9-db4bfaac1757">
<img width="1087" alt="Screenshot 2024-06-04 alle 09 34 38" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/55345633/99c7f137-49cb-4e89-bb40-e943623ce80b">

**Comment by Divine-Sunderer** [comment]
<img width="3093" alt="Fac Simile 27102022  - Cache" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/157913185/3474a1f8-b623-4c07-a395-a89d194f45a9">
